### DISCUSSION TITLE
Ubiquity Flutter's Based Installer

### DISCUSSION THREAD

**Discussion by Reno-Sifana** [linuxmint/discussions#164] [Ideas]
I think is good idea when Linux Mint's Major Upgrades next year (Ubuntu 24.04 LTS based) will Implemented Ubiquity Flutter's Based Installer like Ubuntu 23.04 does. Because i think current Linux Mint Installer's slightly out of date (In My Opinion).
Here's the Screenshot for example
![image](https://github.com/linuxmint/discussions/assets/68702583/001634c0-8399-48a6-8dc2-9d808f672b35)
(Language Selections)
![image](https://github.com/linuxmint/discussions/assets/68702583/6f83ea1f-8e03-4ca2-9266-8b993a293246)
(Select a keyboard layout)
![image](https://github.com/linuxmint/discussions/assets/68702583/aca80747-f2a2-4481-8813-12c2442daa20)
(Connect to Internet)
![image](https://github.com/linuxmint/discussions/assets/68702583/62382f34-88ac-4e0f-9a92-1001c1475a6a)
(Installation options)
![image](https://github.com/linuxmint/discussions/assets/68702583/0e80e77e-265e-4031-8499-b1bb43e33185)
(Install Type)
![image](https://github.com/linuxmint/discussions/assets/68702583/aa69b134-00a5-4f76-ad43-118d87ed97fc)
(Select a Timezone)
![image](https://github.com/linuxmint/discussions/assets/68702583/2c0cf656-7ec8-460a-ae20-77aae747282d)
(Pick an Username and Password)
![image](https://github.com/linuxmint/discussions/assets/68702583/5eadf968-fdb2-4405-bb74-939a89f610d1)
(Installation Progress)
What Do You Think about it?
Thanks.

**Comment by Secret-chest** [comment]
Please don't. Flutter is not OK for GTK desktop apps.

**Reply by Reno-Sifana** [reply]
> Please don't. Flutter is not OK for GTK desktop apps.

Oh I see.

**Comment by Secret-chest** [comment]
In other words, would you use a web browser to partition your disc?

**Comment by anaximeno** [comment]
@Secret-chest Flutter translates the tools (and code created) to native gtk in c++ for linux systems, it is not a web (specifically) framework.
